backend/oasst_backend/config.py

```diff
@@ -44,4 +44,5 @@ def assemble_cors_origins(cls, v: Union[str, List[str]]) -> Union[List[str], str
         raise ValueError(v)
 
 
+print("cats")
```

Comment:
🐈 🐈 🐈

Reply:
that was to force a re-deployment lol.  This is deleted now.